entproto/Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM golang:1.17.3
+FROM golang:1.18.2
```

Comment:
@rotemtam: I need to do some changes in dockerfile/README to able run go generate, can you review it?